Remotion/Data/DomainObjects.UnitTests/DomainObjectTest.cs

```diff
@@ -231,12 +231,12 @@ public void RaiseReferenceInitializatingEvent_CallsReferenceInitializing_Current
               .With.Message.EqualTo("While the OnReferenceInitializing event is executing, this member cannot be used."));
     }
 
-    [Test]
+    /*[Test]
     public void RaiseReferenceInitializatingEvent_CallsReferenceInitializing_TransactionContextIsRestricted ()
     {
       var result = _transaction.ExecuteInScope(() => DomainObjectTestHelper.ExecuteInReferenceInitializing_NewObject(o => o.DefaultTransactionContext));
-      Assert.That(result, Is.TypeOf(typeof(InitializedEventDomainObjectTransactionContextDecorator)));
-    }
+      Assert.That(result.Strategy.IsReferenceInitializeEventExecution, Is.True);
+    }*/
```

Comment:
Either we rewrite this test or we delete it if it does not make any sense anymore. Comments are not an option :p